src/documents/utilities.js

```diff
@@ -122,7 +122,9 @@ export const defaultSort = (a, b) => {
 };
 
 // sort list of file nodes by string similarity to file name
-export const sortFilesBySimilarity = (file, fileList, threshold = 0.3) => {
+export const sortFilesBySimilarity = (file, fileList, threshold = 0.9) => {
```

Comment:
this assignment should use the global constant `STUDY_DOCS_SIMILARITY_THRESHOLD `, if not the constant isn't used

Reply:
We are using the global constants, I have updated their values. And here just adjusting the default threshold value in case we may want to use it in the future.